code/game/objects/items/stacks/money.dm

```diff
@@ -212,6 +239,40 @@
 		value = 4
 		return ..()
 
+/obj/item/stack/money/dollar100
+	name = "100 Dollar Bill"
+	desc = "Paper bank note valued at one-hundred dollars"
```

Comment:
desc = "A paper bank note valued at one-hundred dollars"

Add the "A" for proper flow to the description. If the others don't have it is because I haven't fixed this shitcode yet.